mswindows/osgeo4w/env.bat.tmpl

```diff
@@ -2,8 +2,6 @@ REM
 REM Environmental variables for GRASS OSGeo4W installer
 REM
 
-call "%OSGEO4W_ROOT%\bin\py3_env.bat"

```

Comment:
Why is it not needed anymore?

Reply:
In OSGeo4W next generation  Python2 has been skipped, and Python3 is default; thus no need to provide different python environments